### ISSUE TITLE
Wrong version of org.eclipse.jdt.core used

### ISSUE THREAD

**Issue by pit3k** [revelc/formatter-maven-plugin#258]
The version of `org.eclipse.jdt.core` used in the oxygen release of the plugin is incorrect. It should be 3.13.50.v20171007-0855 instead of 3.13.0. For us this causes formatting of some enums to fail (probably a known bug in older jdt). For example:

```
public enum X {
  ZERO {
    @Override
    public int get() {
      return 0;
    }
  };

  public abstract int get();

}
```

**Comment by ctubbsii** [comment]
I think we were trying to target the "official" artifacts deployed by Eclipse, but it's not clear which ones those are. Their documentation is very poor, and they aren't publishing everything they should be yet.

The version you mention is under a different groupId. Is that what the Eclipse group is publishing as now? It's hard to keep track.

**Comment by pit3k** [comment]
I'm not sure what is the best official source documents listing plugin versions included in given Eclipse release. I've simply checked what is inside freshly installed Eclipse Oxygen, by going to Help -> About -> Details -> Plugins. I'm also not sure if the tycho group is the official one now.

However, I've verified that bumping this version solved the differences we observed between formatting done by eclipse and this plugin.
